app/views/foreman/unattended/kickstart-katello.erb

```diff
@@ -22,7 +22,11 @@ oses:
   section_end = (rhel_compatible && os_major <= 5) ? '' : '%end'
 %>
 install
+<% if host_param('kickstart_liveimg') %>
+liveimg --url=<%= host_param('kickstart_liveimg') %>
```

Comment:
Given https://github.com/theforeman/community-templates/pull/460, I think we should actually limit this PR just to the repository_url actually